routes/api/v1/favorites.js

```diff
@@ -58,7 +58,14 @@ router.get('/:id', async (request, response) => {
   let favorite =  await database('favorites').where('id', request.params.id);
 
   if (favorite.length) {
-    return response.status(200).json(favorite[0]);
+    favoriteTrack = {
+      id: favorite[0].id,
+      title: favorite[0].title,
+      artistName: favorite[0].artistName,
+      genre: favorite[0].genre,
+      rating: favorite[0].rating
+    };
```

Comment:
Nice object literal! That seems like a good way to only retrieve the fields we want. There may be a way to do the same thing in knex but I bet it requires the same number of lines of code.